DiscordBot.Modules/CommandModules/FileExplorerModule.cs

```diff
@@ -28,8 +26,8 @@ public async Task FilesAsync([Remainder] string commandText)
             var filesFolder = configuration["FileStore:DataFolder"];
             if (!Directory.Exists(filesFolder))
             {
-                logger.LogError("FileStore:DataFolder does not refer to an existing directory. Did you configure it in appsettings.json?");
-                await ReplyAsync("Bot not properly configured. Check logs for details.");
+                logger.LogError("FileStore:DataFolder referenziert kein existierendes Verzeichnis. Hast du es in appsettings.json konfiguriert?");
```

Comment:
Why are u guys always germanizing everything? Thats ugly :D

Reply:
@SIMULATAN said, that you want that 😀

Reply:
Thats a lie.